scripts/aws_ecs_benchmarks_cdn.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# make sure the following line is added to `~/.bashrc`
+source "$HOME/.cargo/env"
+
+# assign local ip
+ip=`curl http://169.254.169.254/latest/meta-data/local-ipv4`
+orchestrator_url=http://"$ip":4444
+cdn_marshal_address="$ip":9000
+keydb_address=redis://"$ip":6379
+
+# build to get the bin in advance, uncomment the following if built first time
+# just async_std example validator-push-cdn -- http://localhost:4444 &
+# # remember to sleep enough time if it's built first time
+# sleep 5m
+# for pid in $(ps -ef | grep "validator-push-cdn" | awk '{print $2}'); do sudo kill -9 $pid; done
+
+# docker build and push
+docker build . -f ./docker/validator-cdn.Dockerfile -t ghcr.io/espressosystems/hotshot/validator-webserver:main-async-std
```

Comment:
Do we want to be tagging the CDN validator with the webserver tag? This might be a source of confusion in the future

Reply:
If you add it to the CI it should just create one for you !

Reply:
Added to the CI, and will change the tag (in the next PR after this is merged in) once the image is there and I test it with it successfully.